robosuite/scripts/demo_device_control.py

```diff
@@ -151,6 +153,8 @@
         ignore_done=True,
         use_camera_obs=False,
         gripper_visualizations=True,
+        #render_collision_mesh=True,
```

Comment:
remove comment

Reply:
follow up from above - I can guarantee you that I did not touch this script at all

Reply:
fixed